lib/bundler/cli/gem.rb

```diff
@@ -124,6 +124,15 @@ def run
         templates.merge!("CODE_OF_CONDUCT.md.tt" => "CODE_OF_CONDUCT.md")
       end
 
+      if ask_and_set(:rubocop, "Do you want to add rubocop as a dependency for gems you generate?",
+        "RuboCop is a static code analyzer that has out-of-the-box rules for many " \
+        "of the guidelines in the community style guide. " \
+        "For more information, see the RuboCop docs (http://www.rubocop.org/en/latest/) " \
```

Comment:
The RuboCop docs URL has been updated to https://docs.rubocop.org.

cf. https://github.com/rubocop-hq/rubocop/commit/86b4a41e59fdba6fbd70b26a0fc0f1a2c916c56b#diff-458bc38d82061cdac508dfb1be2b918b